azurelinuxagent/common/protocol/imds.py

```diff
@@ -11,7 +11,7 @@
 from azurelinuxagent.common.utils.flexible_version import FlexibleVersion
 
 IMDS_ENDPOINT = '169.254.169.254'
-APIVERSION = '2017-12-01'
+APIVERSION = '2018-02-01'
```

Comment:
This version is not available in sovereign clouds nor was the previous version.

/cc @harijayms

Reply:
Hari asked me to use this version.

Reply:
I havent updated the docs, but this version should be available now in national clouds.